### PR TITLE
SiteMap.html  Enhancement

### LINKED ISSUE #278: SiteMap Enhancement
The site map of  the project is not so good looking . Even the arrows are not merging properly and in some places arrows defining the map are not present .

@harikesh409 , I want to fix this issue , please assign me this issue , so that I can proceed.

### PR DESCRIPTION
**Pull request by tier3guy**
Fixes #278

In this PR I made some changes in UI and as the major issue was that the arrows were not aligned properly , so I fixed that . Please do consider my PR.

After changes :

### CODE REVIEW

File: css/sitemapStyles.css [hunk 1]
```diff
@@ -1,90 +1,119 @@
...
+    padding : 0;
+    font-family: Poppins;
 }
 body{ 
-    background:#001d1a; 
-    max-width:980px; 
-    margin:0 auto; 
-    padding:5px; 
-    font: 14px "Arial", Tahoma, sans-serif; 
+    background:#000000;
```
Comment (harikesh409) [inline css/sitemapStyles.css:10]: Don't use black background it is never recommended. Choose the existing color scheme.

File: css/sitemapStyles.css [hunk 2]
```diff
@@ -1,90 +1,119 @@
...
+
+.heading{
+    display: flex;
+    align-items: center;
+    padding: 15px 12px;
+    font-size: 2.5rem;
+
+
 }
+
```
Comment (harikesh409) [inline css/sitemapStyles.css:22]: Remove unnecessary blank lines.

File: sitemap.html [hunk 3]
```diff
@@ -1,33 +1,61 @@
...
-			<div><a href="https://algo-phantoms.github.io/Algo-Phantoms-Frontend/#about">About Page</a></div>
 
-			<div><a href="https://algo-phantoms.github.io/Algo-Phantoms-Frontend/team.html">Teams Page</a></div>
-			<div><a href="https://algo-phantoms.github.io/Algo-Phantoms-Frontend/#footer">Contact Page</a></div>
-			<div><a href="https://www.codechef.com/ide">Code Editor</a></div>
-			<div><a href="https://www.geeksforgeeks.org/data-structure-gq/">Quizes</a></div>
-			<div><a href="https://github.com/Algo-Phantoms/Algo-Tree">DSA Guidance</a></div>
-			<div><a href="https://github.com/Algo-Phantoms">GitHub</a></div>	
+			<div class="main">
+			<div class="links">
```
Comment (harikesh409) [inline sitemap.html:32]: Use proper indentations.

File: sitemap.html [hunk 4]
```diff
@@ -1,33 +1,61 @@
...
+
+
+				<div class="features">
+					<a href="https://www.codechef.com/ide">Code Editor</a>
+					<a href="https://www.geeksforgeeks.org/data-structure-gq/">Quizes</a>
+				</div>
+
+
+				<div class="contact">
+					<a href="https://github.com/Algo-Phantoms/Algo-Tree">DSA Guidance</a>
```
Comment (harikesh409) [inline sitemap.html:48]: Instead of pointing to the github repo point to the `DSA.html`.

**Review by harikesh409 — CHANGES_REQUESTED** [review]

**Comment by tier3guy** [comment]
@harikesh409 sir , I have already made changes but in my sitemap branch the changes are implemented successfully but not in the master branch even after the merging

**Comment by harikesh409** [comment]
Moved to #310

**Pull request closed without merging**